polysnapper.js

```diff
@@ -205,19 +205,37 @@ function PolySnapper(opts){
                 endLat   = swLatlng.lat();
                 startLng = swLatlng.lng();
 
-                lat = startLat + (( e.offsetY/ this.offsetHeight ) * (endLat - startLat));
-                lng = startLng + (( e.offsetX/ this.offsetWidth )  * (endLng - startLng));
+                fullscreenMode = function () {
+                    //Verifies if map is in fullscreen mode. If so, its size will match window's size
+                    if ($(_map.getDiv()).children().eq(0).height() == window.innerHeight &&
```

Comment:
I used the verification described on [this StackOverflow question](https://stackoverflow.com/questions/39620850/google-maps-api-v3-how-to-detect-when-map-changes-to-full-screen-mode) because seems to be the easier to implement. Couldn't find a better way to check the fullscreen mode for the map (where I'm using is necessary know if the screen is in fullscreen or only the map).